docs/proposals/formatted-output.md

```diff
@@ -379,6 +379,105 @@ oras discover $REGISTRY/$REPO:v1 --format json
 > [!NOTE]
 > The `--format` flag will replace the existing `--output` flag. The `--output` will be marked as "deprecated" in ORAS v1.2.0 and will be removed in the future releases.
 
+If the referrers have associated referrers, ORAS should be able to show the manifest content of the subject image and all referrers. 
+
+When showing the subject image and all referrers' manifests recursively in a tree view output, the following fields should be returned:
+
+- `reference`: full reference by digest of the subject image
+- `mediaType`: media type of the subject image
+- `digest`: digest of the subject image
+- `size`: subject image size in bytes
+- `manifests`: the list of referrers' manifest
+  - `reference`: full reference by digest of the referrer
+  - `mediaType`: media type of the referrer
+  - `digest`: digest of the referrer
+  - `size`: referrer file size in bytes
+  - `artifactType`: artifact type of a referrer
+  - `annotations`: contains arbitrary metadata in a referrer
+  - `referrerManifests`: the list of referrers' manifest
```

Comment:
What about `manifests` or `referrers`

Reply:
In terms of consistency, `manifests` actually makes more sense as the schema is the same, just that users may not understand what does "manifests" mean. Maybe we should have called the root manifests `referrers` in the first place😂...

Reply:
We will use `referrers` as the field name in those two places based on the community discussion.